### DISCUSSION TITLE
In summer 2023 the LSC team has set up this discussion board

### DISCUSSION THREAD

**Discussion by pvgenuchten** [lsc-hubs/kenya-hub#1] [Announcements]
We love to hear your ideas and thoughts about the potential and risks for the LSC-hub Kenya.

**Comment by Ermias-Betemariam** [comment]
Great progress! Any chance to use administrative boundaries to clip raster datasets?

**Reply by pvgenuchten** [reply]
Ermias, thank you for the suggestion, can you explain where this procedure could be implemented, my colleague Maria may be able to share a R script which facilitates a clipping process based on administrative boundaries
